src/backend/marsha/urls.py

```diff
@@ -54,3 +58,6 @@
     urlpatterns += [
         path("development/", DevelopmentLTIView.as_view(), name="lti-development-view")
     ]
+    # User facing site is hidden behind the DEBUG flag until it is ready to
```

Comment:
I don't understand this comment. How the `DEBUG` flag is acting here ? I think there is something better to use.in #800 we installed [waffle](https://waffle.readthedocs.io/en/stable/index.html) and there is a feature to enable an entire view with a feature flag. What do you thin about using this ?

Reply:
Yes, this is perfect for my use-case, thank you 👍